test/e2e/specs/editor/various/draggable-blocks.spec.js

```diff
@@ -0,0 +1,158 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'Draggable block', () => {
+	test.beforeEach( async ( { admin, page } ) => {
+		await admin.createNewPost();
+		// Make the viewport large enough so that a scrollbar isn't displayed.
+		// Otherwise, the page scrolling can interfere with the test runner's
+		// ability to drop a block in the right location.
+		await page.setViewportSize( {
+			width: 960,
+			height: 1024,
+		} );
```

Comment:
TIL that we can use viewport size for whole spec using `test.use`:

```
test.use({
  viewport: { width: 960, height: 1024 },
});
```

See https://github.com/microsoft/playwright/blob/accf5b043d01a7860aa4b863bf2eadcce57402ca/docs/src/emulation.md#viewport.